net/keepalived/files/etc/keepalived/scripts/rsync.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
```

Comment:
I would suggest to move this scripts to` /use/share/keepalived` or an other place and not to `/etc`

Reply:
so in that case wouldn't It needs to be added to conf files so that it can be backed up?

Reply:
The question is rather do you want the file to be taken along with a system upgrade, or should it be installed by the package. As I see it, the script belongs to the `keepalived-sync` package and not to the runtime config.

Where is the rsync.sh used at all?

Reply:
https://github.com/openwrt/packages/pull/19374#discussion_r974193740

Reply:
Got you, moved `/etc/keepalived/scripts/rsync.sh` to `/usr/share/keepalived/scripts/rsync.sh` and created soft link to `/etc/keepalived/scripts/rsync.sh` from the Makefile